src/test/java/com/diffplug/gradle/eclipse/MavenCentralMappingTest.java

```diff
@@ -42,7 +42,23 @@ private void assert463(Map<String, String> bundleToVersion) {
 		Assertions.assertThat(bundleToVersion)
 				.containsEntry("org.eclipse.debug.core", "3.10.100")
 				.containsEntry("org.eclipse.equinox.p2.metadata", "2.3.100")
-				.hasSize(895);
+				.containsEntry("org.eclipse.help", "3.7.0")
+				.hasSize(743);
```

Comment:
`743` is also the size returned bnd when listing the content of the
`https://download.eclipse.org/eclipse/updates/4.6/`

Output: https://github.com/jmini/ecentral/blob/master/data/4.6/bnd-output.txt

---

Approach described on stackoverflow:
[How to programmatically query a p2 repository for information and artifacts?](https://stackoverflow.com/a/54445914/91497)